php-checks/src/main/resources/org/sonar/l10n/php/rules/php/S1116.json

```diff
@@ -1,16 +1,16 @@
 {
   "title": "Empty statements should be removed",
-  "type": "BUG",
+  "type": "CODE_SMELL",
```

Comment:
The commit message does not describe that change (same thing for tags and for other rules).